packages/tasit-sdk/src/helpers/DecentralandUtils.test.js

```diff
@@ -0,0 +1,81 @@
+import { Action } from "../TasitSdk";
+const { ConfigLoader, ERC20, ERC721, Marketplace } = Action;
+const { Mana } = ERC20;
+const { Estate, Land } = ERC721;
+const { Decentraland } = Marketplace;
+import DecentralandUtils from "./DecentralandUtils";
+
+// Note: This test suite assumes that all parcels and estates of SellerWallet are for sale
+// See more: `TasitSDk/packages/tasit-contracts/src/script/populateDecentralandContracts.js`
+describe("DecentralandUtils", () => {
+  const [minterWallet, sellerWallet] = accounts;
+  const { address: sellerAddress } = sellerWallet;
+  let mana;
+  let land;
+  let estate;
+  let marketplace;
+  const decentralandUtils = new DecentralandUtils();
+  const {
+    getAllAssetsForSale,
+    getEstateIdsOf,
+    getParcelIdsOf,
+    getAssetsOf,
+  } = decentralandUtils;
+  let assetsForSale;
+
+  const isAssetAnEstate = asset => asset.nftAddress === estate.getAddress();
+  const isAssetAParcel = asset => asset.nftAddress === land.getAddress();
+
+  beforeEach("", async () => {
+    mana = new Mana(MANA_ADDRESS);
+    land = new Land(LAND_PROXY_ADDRESS);
+    estate = new Estate(ESTATE_ADDRESS);
+    marketplace = new Decentraland(MARKETPLACE_ADDRESS);
+
+    assetsForSale = await getAllAssetsForSale();
+  });
+
+  it("should get parcels for sale", async () => {
+    const parcelsForSale = assetsForSale.filter(isAssetAParcel);
+    const { length: parcelsForSaleAmount } = parcelsForSale;
+
+    const parcelsOfSeller = await land.balanceOf(sellerAddress);
+
+    expect(`${parcelsOfSeller}`).to.equal(`${parcelsForSaleAmount}`);
+  });
+
+  it("should get estates for sale", async () => {
+    const estatesForSale = assetsForSale.filter(isAssetAnEstate);
+    const { length: estatesForSaleAmount } = estatesForSale;
+
+    const estatesOfSeller = await estate.balanceOf(sellerAddress);
+
+    expect(`${estatesOfSeller}`).to.equal(`${estatesForSaleAmount}`);
+  });
+
+  it("should get assets owned by an address", async () => {
```

Comment:
Should we write a new test where it's not asserting on the seller but instead a buyer?

Reply:
To test a `no asset` case?

Reply:
I guess an functional test of this type to cover the most common in-app flow - a user has no land but some MANA. The user buys one piece of land. Now the user indeed has 1 piece of land as reported by the "getMyAssets" (I forget the exact name) function like we would expect.

Reply:
A `no asset` case is potentially interesting too to make sure the function doesn't error out

Reply:
Done.

Reply:
Great, thanks! Resolving this.